### PR TITLE
js_notification_DOM

### PR DESCRIPTION
**Pull request by mykolahuryn**
[DEMO LINK](https://mykolahuryn.github.io/js_notification_DOM/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,7 +1,27 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector('body');
+  const div = document.createElement('div');
+  const h2 = document.createElement('h2');
+  const p = document.createElement('p');
```
Comment (slavanosatskyi) [inline src/scripts/main.js:7]: Would be better to provide more meaningful names here - notification, header, content

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,7 +1,27 @@
...
+
+  h2.textContent = title;
+  p.textContent = description;
+
+  div.classList.add('notification', type);
+  h2.classList.add('title');
+
+  div.style.position = 'absolute';
+  div.style.top = posTop + 'px';
+  div.style.right = posRight + 'px';
```
Comment (slavanosatskyi) [inline src/scripts/main.js:20]: as an option you can use here string templates

**Review by slavanosatskyi — CHANGES_REQUESTED** [review]
Nice, though there are a couple of changes needed

**Review by witflash — APPROVED** [review]
Much better!
Well done 👍🏻

**Pull request closed without merging**